pkg/cloud/gcp/gcs_storage_test.go

```diff
@@ -173,9 +173,9 @@ func TestCompressedGCS(t *testing.T) {
 	conf2, err := cloud.ExternalStorageConfFromURI(gsFile2, user)
 	require.NoError(t, err)
 
-	s1, err := cloud.MakeExternalStorage(ctx, conf1, base.ExternalIODirConfig{}, testSettings, nil, nil, nil)
+	s1, err := cloud.MakeExternalStorage(ctx, conf1, base.ExternalIODirConfig{}, testSettings, nil, nil, nil, nil)
```

Comment:
would be nice to `require.True(t, s1.RequiresExternalIOAccounting())` (in other implementation tests too)